### PR TITLE
Build mod_md for apache-24

### PR DESCRIPTION
**Pull request by sjorge**
https://httpd.apache.org/docs/2.4/mod/mod_md.html

This is pretty nice when grabbing your certificates via letsencrypt or another acme compatible CA.

@citrus-it this is the thing I was blabbering about on IRC.

I'm not super excited about having it pull in libjansson and curl as dependencies though. I there an easy way to split mod_md off to a separate package? Or would that be more work than it's worth?

### CODE REVIEW

File: build/libjansson/build.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+PROG=libjansson
+VER=2.13.1
+PKG=ooce/library/libjansson
+SUMMARY="libjansson"
+DESC="C library for encoding, decoding and manipulating JSON data"
+
+BUILDDIR="jansson-$VER"
+
+set_mirror 'https://github.com/akheron/jansson/archive/refs/tags/'
```
Comment (sjorge) [inline build/libjansson/build.sh:27]: I guess this should somehow be added to the omnios source mirror so these lines can go?

Comment (citrus-it) [reply build/libjansson/build.sh:27]: It's on our mirror now as https://mirrors.omnios.org/jansson/jansson-2.13.1.tar.bz2 so, yes, please drop these lines.

File: build/libjansson/build.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
```
[the remark this answers is not in the document]
Comment (sjorge) [reply build/libjansson/build.sh]: Done, also fixed up the one in local.mog

**Comment by sjorge** [comment]
I also updated cyrus to no longer build jansson as a bundled library and instead use the package.

**Review by citrus-it — APPROVED** [review]

**Review by citrus-it — APPROVED** [review]

**Review by hadfl — APPROVED** [review]

**Pull request merged**